### PR TITLE
Upgrade ktlint conventions

### PR DESCRIPTION
**Pull request by eskatos**
Upgrade `ktlint-conventions` plugin that brings latest `ktlint-gradle` plugin and `ktlint`.

Existing rules got more checks in ktlint. This pull request also applies the new rules to all sources creating a hugs diff with lots of whitespace changes. Better look at individual commits to see the changes to the build.

### CODE REVIEW

**Comment by eskatos** [comment]
@bot-gradle test this

File: subprojects/kotlin-dsl-plugins/src/integTest/kotlin/org/gradle/kotlin/dsl/plugins/embedded/EmbeddedKotlinPluginTest.kt [hunk 1]
```diff
@@ -208,15 +218,17 @@ class EmbeddedKotlinPluginTest : AbstractPluginTest() {
...
+            """
 
             plugins {
                 `embedded-kotlin`
             }
 
             $repositoriesBlock
 
-        """)
+        """
```
Comment (bamboo) [inline subprojects/kotlin-dsl-plugins/src/integTest/kotlin/org/gradle/kotlin/dsl/plugins/embedded/EmbeddedKotlinPluginTest.kt:230]: ❌  indentation looks off

File: subprojects/kotlin-dsl-plugins/src/integTest/kotlin/org/gradle/kotlin/dsl/plugins/precompiled/PrecompiledScriptPluginAccessorsTest.kt [hunk 2]
```diff
@@ -79,41 +81,54 @@ class PrecompiledScriptPluginAccessorsTest : AbstractPrecompiledScriptPluginTest
                             "theGradleProperty",
                             property
                         )
-                    """)
+                    """
```
Comment (bamboo) [inline subprojects/kotlin-dsl-plugins/src/integTest/kotlin/org/gradle/kotlin/dsl/plugins/precompiled/PrecompiledScriptPluginAccessorsTest.kt:84]: ❌  indentation looks off

File: subprojects/kotlin-dsl/src/testFixtures/kotlin/org/gradle/kotlin/dsl/fixtures/AbstractKotlinIntegrationTest.kt [hunk 3]
```diff
@@ -104,21 +104,27 @@ abstract class AbstractKotlinIntegrationTest : AbstractIntegrationTest() {
...
+        withBuildScriptIn(
+            "buildSrc",
+            """
             plugins {
                 `kotlin-dsl`
             }
 
             $repositoriesBlock
-        """)
+        """
```
Comment (bamboo) [inline subprojects/kotlin-dsl/src/testFixtures/kotlin/org/gradle/kotlin/dsl/fixtures/AbstractKotlinIntegrationTest.kt:126]: ❌  indentation looks off

File: subprojects/kotlin-dsl/src/testFixtures/kotlin/org/gradle/kotlin/dsl/fixtures/AbstractKotlinIntegrationTest.kt [hunk 4]
```diff
@@ -104,21 +104,27 @@ abstract class AbstractKotlinIntegrationTest : AbstractIntegrationTest() {
...
     protected
     fun withBuildSrc() =
-        withFile("buildSrc/src/main/groovy/build/Foo.groovy", """
+        withFile(
+            "buildSrc/src/main/groovy/build/Foo.groovy",
+            """
             package build
             class Foo {}
-        """)
+        """
```
Comment (bamboo) [inline subprojects/kotlin-dsl/src/testFixtures/kotlin/org/gradle/kotlin/dsl/fixtures/AbstractKotlinIntegrationTest.kt:112]: ❌  indentation looks off

File: subprojects/kotlin-dsl/src/test/kotlin/org/gradle/kotlin/dsl/execution/ResidualProgramCompilerTest.kt [hunk 5]
```diff
@@ -398,11 +398,13 @@ class ResidualProgramCompilerTest : TestWithCompiler() {
...
 
         val source = ProgramSource(
-            "build.gradle.kts", """
+            "build.gradle.kts",
+            """
             buildscript { println("stage 1 buildscript") }
             plugins { println("stage 1 plugins") }
             print("stage 2")
-        """.replaceIndent())
+        """.replaceIndent()
```
Comment (bamboo) [inline subprojects/kotlin-dsl/src/test/kotlin/org/gradle/kotlin/dsl/execution/ResidualProgramCompilerTest.kt:406]: ❌  indentation looks off

File: subprojects/kotlin-dsl/src/test/kotlin/org/gradle/kotlin/dsl/execution/ResidualProgramCompilerTest.kt [hunk 6]
```diff
@@ -449,10 +451,12 @@ class ResidualProgramCompilerTest : TestWithCompiler() {
     fun `Static(ApplyPluginRequestsOf(Stage1Sequence(pluginManagement, plugins))`() {
         // Given
         val source = ProgramSource(
-            "settings.gradle.kts", """
+            "settings.gradle.kts",
+            """
             pluginManagement { println("stage 1 pluginManagement") }
             plugins { println("stage 1 plugins") }
-        """.replaceIndent())
+        """.replaceIndent()
```
Comment (bamboo) [inline subprojects/kotlin-dsl/src/test/kotlin/org/gradle/kotlin/dsl/execution/ResidualProgramCompilerTest.kt:458]: ❌  indentation looks off

File: subprojects/kotlin-dsl/src/test/kotlin/org/gradle/kotlin/dsl/execution/ProgramParserTest.kt [hunk 7]
```diff
@@ -72,32 +75,38 @@ class ProgramParserTest {
...
 
         val source = ProgramSource(
-            "build.gradle.kts", """
+            "build.gradle.kts",
+            """
             buildscript { println("stage 1 buildscript") }
             plugins { println("stage 1 plugins") }
             print("stage 2")
-        """.replaceIndent())
+        """.replaceIndent()
```
Comment (bamboo) [inline subprojects/kotlin-dsl/src/test/kotlin/org/gradle/kotlin/dsl/execution/ProgramParserTest.kt:108]: ❌  indentation looks off

File: subprojects/kotlin-dsl/src/test/kotlin/org/gradle/kotlin/dsl/execution/ProgramParserTest.kt [hunk 8]
```diff
@@ -141,19 +153,22 @@ class ProgramParserTest {
...
         val source = ProgramSource(
-            "settings.gradle.kts", """
+            "settings.gradle.kts",
+            """
             pluginManagement { println("stage 1 pluginManagement") }
             buildscript { println("stage 1 buildscript") }
             plugins { println("stage 1 plugins") }
             print("stage 2")
-        """.replaceIndent())
+        """.replaceIndent()
```
Comment (bamboo) [inline subprojects/kotlin-dsl/src/test/kotlin/org/gradle/kotlin/dsl/execution/ProgramParserTest.kt:170]: ❌  indentation looks off

File: subprojects/kotlin-dsl/src/test/kotlin/org/gradle/kotlin/dsl/execution/ProgramParserTest.kt [hunk 9]
```diff
@@ -191,7 +208,8 @@ class ProgramParserTest {
             dependencies {
                 implementation("org.acme:lib:1.0")
             }
-        """)
+        """
```
Comment (bamboo) [inline subprojects/kotlin-dsl/src/test/kotlin/org/gradle/kotlin/dsl/execution/ProgramParserTest.kt:211]: ❌  indentation looks off

File: subprojects/kotlin-dsl/src/test/kotlin/org/gradle/kotlin/dsl/accessors/PluginAccessorsClassPathTest.kt [hunk 10]
```diff
@@ -137,7 +138,8 @@ class PluginAccessorsClassPathTest : TestWithClassPath() {
                      */
                     val `MyOwnPluginGroup`.`plugin`: PluginDependencySpec
                         get() = plugins.id("my.own.plugin")
-            """)
+            """
```
Comment (bamboo) [inline subprojects/kotlin-dsl/src/test/kotlin/org/gradle/kotlin/dsl/accessors/PluginAccessorsClassPathTest.kt:141]: ❌  indentation looks off

File: subprojects/kotlin-dsl/src/test/kotlin/org/gradle/kotlin/dsl/NamedContainersDslTest.kt [hunk 11]
```diff
@@ -158,7 +172,8 @@ class NamedContainersDslTest : AbstractDslTest() {
                     extendsFrom(cabin.get())
                 }
             }
-        """) {
+        """
```
Comment (bamboo) [inline subprojects/kotlin-dsl/src/test/kotlin/org/gradle/kotlin/dsl/NamedContainersDslTest.kt:175]: ❌  indentation looks off

File: subprojects/kotlin-dsl/src/test/kotlin/org/gradle/kotlin/dsl/NamedContainersDslTest.kt [hunk 12]
```diff
@@ -139,13 +150,16 @@ class NamedContainersDslTest : AbstractDslTest() {
                     extendsFrom(valley.get())
                 }
             }
-        """)
+        """
```
Comment (bamboo) [inline subprojects/kotlin-dsl/src/test/kotlin/org/gradle/kotlin/dsl/NamedContainersDslTest.kt:153]: ❌  indentation looks off

File: subprojects/kotlin-dsl/src/test/kotlin/org/gradle/kotlin/dsl/NamedContainersDslTest.kt [hunk 13]
```diff
@@ -110,13 +118,16 @@ class NamedContainersDslTest : AbstractDslTest() {
             val hill: NamedDomainObjectProvider<Configuration> by configurations.registering {
                 extendsFrom(valley.get())
             }
-        """)
+        """
```
Comment (bamboo) [inline subprojects/kotlin-dsl/src/test/kotlin/org/gradle/kotlin/dsl/NamedContainersDslTest.kt:121]: ❌  indentation looks off

File: subprojects/kotlin-dsl/src/test/kotlin/org/gradle/kotlin/dsl/NamedContainersDslTest.kt [hunk 14]
```diff
@@ -83,13 +88,16 @@ class NamedContainersDslTest : AbstractDslTest() {
                     extendsFrom(valley.get())
                 }
             }
-        """)
+        """
```
Comment (bamboo) [inline subprojects/kotlin-dsl/src/test/kotlin/org/gradle/kotlin/dsl/NamedContainersDslTest.kt:91]: ❌  indentation looks off

File: subprojects/kotlin-dsl/src/test/kotlin/org/gradle/kotlin/dsl/NamedContainersDslTest.kt [hunk 15]
```diff
@@ -54,13 +56,16 @@ class NamedContainersDslTest : AbstractDslTest() {
             val hill: NamedDomainObjectProvider<Configuration> = configurations.register("hill") {
                 extendsFrom(valley.get())
             }
-        """)
+        """
```
Comment (bamboo) [inline subprojects/kotlin-dsl/src/test/kotlin/org/gradle/kotlin/dsl/NamedContainersDslTest.kt:59]: ❌  indentation looks off

File: subprojects/kotlin-dsl/src/integTest/kotlin/org/gradle/kotlin/dsl/support/ImplicitImportsTest.kt [hunk 16]
```diff
@@ -28,13 +30,15 @@ class ImplicitImportsTest : AbstractKotlinIntegrationTest() {
...
-        withBuildScript("""
+        withBuildScript(
+            """
             val a = Callable { "some" }
             val b = TimeUnit.DAYS
             val c = File("some")
             val d = BigDecimal.ONE
             val e = BigInteger.ONE
-        """)
+        """
```
Comment (bamboo) [inline subprojects/kotlin-dsl/src/integTest/kotlin/org/gradle/kotlin/dsl/support/ImplicitImportsTest.kt:40]: ❌  indentation looks off

File: subprojects/kotlin-dsl/src/integTest/kotlin/org/gradle/kotlin/dsl/support/ImplicitImportsTest.kt [hunk 17]
```diff
@@ -12,11 +12,13 @@ class ImplicitImportsTest : AbstractKotlinIntegrationTest() {
...
 
         // given:
-        withBuildScript("""
+        withBuildScript(
+            """
 
             println("*" + Jar::class.qualifiedName + "*")
 
-        """)
+        """
```
Comment (bamboo) [inline subprojects/kotlin-dsl/src/integTest/kotlin/org/gradle/kotlin/dsl/support/ImplicitImportsTest.kt:20]: ❌  indentation looks off

File: subprojects/kotlin-dsl/src/integTest/kotlin/org/gradle/kotlin/dsl/support/EmbeddedKotlinProviderTest.kt [hunk 18]
```diff
@@ -89,7 +96,8 @@ class EmbeddedKotlinProviderTest : AbstractKotlinIntegrationTest() {
                     }
                 }
             }
-        """)
+        """
```
Comment (bamboo) [inline subprojects/kotlin-dsl/src/integTest/kotlin/org/gradle/kotlin/dsl/support/EmbeddedKotlinProviderTest.kt:99]: ❌  indentation looks off

File: subprojects/kotlin-dsl/src/integTest/kotlin/org/gradle/kotlin/dsl/support/EmbeddedKotlinProviderTest.kt [hunk 19]
```diff
@@ -63,14 +67,16 @@ class EmbeddedKotlinProviderTest : AbstractKotlinIntegrationTest() {
...
+        withBuildScript(
+            """
             buildscript {
                 $repositoriesBlock
                 dependencies {
                     classpath("org.jetbrains.kotlin:kotlin-compiler-embeddable:1.3.31")
                 }
             }
-        """)
+        """
```
Comment (bamboo) [inline subprojects/kotlin-dsl/src/integTest/kotlin/org/gradle/kotlin/dsl/support/EmbeddedKotlinProviderTest.kt:78]: ❌  indentation looks off

File: subprojects/kotlin-dsl/src/integTest/kotlin/org/gradle/kotlin/dsl/support/EmbeddedKotlinProviderTest.kt [hunk 20]
```diff
@@ -47,11 +49,13 @@ class EmbeddedKotlinProviderTest : AbstractKotlinIntegrationTest() {
...
     @ToBeFixedForConfigurationCache(because = ":buildEnvironment")
     fun `stdlib and reflect are pinned to the embedded kotlin version for requested plugins`() {
-        withBuildScript("""
+        withBuildScript(
+            """
             plugins {
                 kotlin("jvm") version "1.3.31"
             }
-        """)
+        """
```
Comment (bamboo) [inline subprojects/kotlin-dsl/src/integTest/kotlin/org/gradle/kotlin/dsl/support/EmbeddedKotlinProviderTest.kt:57]: ❌  indentation looks off

File: subprojects/kotlin-dsl/src/integTest/kotlin/org/gradle/kotlin/dsl/support/EmbeddedKotlinProviderTest.kt [hunk 21]
```diff
@@ -27,15 +27,17 @@ class EmbeddedKotlinProviderTest : AbstractKotlinIntegrationTest() {
...
+            """
             buildscript {
                 $repositoriesBlock
                 dependencies {
                     classpath("org.jetbrains.kotlin:kotlin-stdlib:1.0")
                     classpath("org.jetbrains.kotlin:kotlin-reflect:1.0")
                 }
             }
-        """)
+        """
```
Comment (bamboo) [inline subprojects/kotlin-dsl/src/integTest/kotlin/org/gradle/kotlin/dsl/support/EmbeddedKotlinProviderTest.kt:39]: ❌  indentation looks off

File: subprojects/kotlin-dsl/src/integTest/kotlin/org/gradle/kotlin/dsl/resolver/SourceDistributionResolverIntegrationTest.kt [hunk 22]
```diff
@@ -20,7 +21,8 @@ class SourceDistributionResolverIntegrationTest : AbstractKotlinIntegrationTest(
                 "Expected source directories but got none"
             }
 
-        """)
+        """
```
Comment (bamboo) [inline subprojects/kotlin-dsl/src/integTest/kotlin/org/gradle/kotlin/dsl/resolver/SourceDistributionResolverIntegrationTest.kt:24]: ❌  indentation looks off

File: subprojects/kotlin-dsl/src/integTest/kotlin/org/gradle/kotlin/dsl/resolver/KotlinScriptDependenciesResolverTest.kt [hunk 23]
```diff
@@ -355,13 +378,18 @@ class KotlinScriptDependenciesResolverTest : AbstractKotlinIntegrationTest() {
...
+        """
+        )
         withBuildScript("")
-        withBuildScriptIn("a", """
+        withBuildScriptIn(
+            "a",
+            """
             configurations.getByName("doNotExists")
-        """)
+        """
```
Comment (bamboo) [inline subprojects/kotlin-dsl/src/integTest/kotlin/org/gradle/kotlin/dsl/resolver/KotlinScriptDependenciesResolverTest.kt:391]: ❌  indentation looks off

File: subprojects/kotlin-dsl/src/integTest/kotlin/org/gradle/kotlin/dsl/resolver/KotlinScriptDependenciesResolverTest.kt [hunk 24]
```diff
@@ -355,13 +378,18 @@ class KotlinScriptDependenciesResolverTest : AbstractKotlinIntegrationTest() {
     fun `report file warning on runtime failure in another script`() {
         assumeNonEmbeddedGradleExecuter()
 
-        withDefaultSettings().appendText("""
+        withDefaultSettings().appendText(
+            """
             include("a", "b")
-        """)
+        """
```
Comment (bamboo) [inline subprojects/kotlin-dsl/src/integTest/kotlin/org/gradle/kotlin/dsl/resolver/KotlinScriptDependenciesResolverTest.kt:384]: ❌  indentation looks off

File: subprojects/kotlin-dsl/src/integTest/kotlin/org/gradle/kotlin/dsl/resolver/KotlinScriptDependenciesResolverTest.kt [hunk 25]
```diff
@@ -334,12 +352,17 @@ class KotlinScriptDependenciesResolverTest : AbstractKotlinIntegrationTest() {
...
             ${EditorReports.locationAwareEditorHintsPropertyName}=true
-        """)
-        val editedScript = withBuildScript("""
+        """
+        )
+        val editedScript = withBuildScript(
+            """
             configurations.getByName("doNotExists")
-        """)
+        """
```
Comment (bamboo) [inline subprojects/kotlin-dsl/src/integTest/kotlin/org/gradle/kotlin/dsl/resolver/KotlinScriptDependenciesResolverTest.kt:364]: ❌  indentation looks off

File: subprojects/kotlin-dsl/src/integTest/kotlin/org/gradle/kotlin/dsl/resolver/KotlinScriptDependenciesResolverTest.kt [hunk 26]
```diff
@@ -334,12 +352,17 @@ class KotlinScriptDependenciesResolverTest : AbstractKotlinIntegrationTest() {
...
         assumeNonEmbeddedGradleExecuter()
 
-        withFile("gradle.properties", """
+        withFile(
+            "gradle.properties",
+            """
             ${EditorReports.locationAwareEditorHintsPropertyName}=true
-        """)
-        val editedScript = withBuildScript("""
+        """
```
Comment (bamboo) [inline subprojects/kotlin-dsl/src/integTest/kotlin/org/gradle/kotlin/dsl/resolver/KotlinScriptDependenciesResolverTest.kt:359]: ❌  indentation looks off

File: subprojects/kotlin-dsl/src/integTest/kotlin/org/gradle/kotlin/dsl/resolver/KotlinScriptDependenciesResolverTest.kt [hunk 27]
```diff
@@ -316,9 +332,11 @@ class KotlinScriptDependenciesResolverTest : AbstractKotlinIntegrationTest() {
     fun `report file warning on runtime failure in currently edited script`() {
         assumeNonEmbeddedGradleExecuter()
 
-        val editedScript = withBuildScript("""
+        val editedScript = withBuildScript(
+            """
             configurations.getByName("doNotExists")
-        """)
+        """
```
Comment (bamboo) [inline subprojects/kotlin-dsl/src/integTest/kotlin/org/gradle/kotlin/dsl/resolver/KotlinScriptDependenciesResolverTest.kt:338]: ❌  indentation looks off

File: subprojects/kotlin-dsl/src/integTest/kotlin/org/gradle/kotlin/dsl/resolver/KotlinScriptDependenciesResolverTest.kt [hunk 28]
```diff
@@ -293,13 +304,18 @@ class KotlinScriptDependenciesResolverTest : AbstractKotlinIntegrationTest() {
...
+        )
         withBuildScript("")
-        withBuildScriptIn("a", """
+        withBuildScriptIn(
+            "a",
+            """
             doNotExists()
-        """)
+        """
```
Comment (bamboo) [inline subprojects/kotlin-dsl/src/integTest/kotlin/org/gradle/kotlin/dsl/resolver/KotlinScriptDependenciesResolverTest.kt:317]: ❌  indentation looks off

File: subprojects/kotlin-dsl/src/integTest/kotlin/org/gradle/kotlin/dsl/resolver/KotlinScriptDependenciesResolverTest.kt [hunk 29]
```diff
@@ -293,13 +304,18 @@ class KotlinScriptDependenciesResolverTest : AbstractKotlinIntegrationTest() {
     fun `report file warning on script compilation failure in another script`() {
         assumeNonEmbeddedGradleExecuter()
 
-        withDefaultSettings().appendText("""
+        withDefaultSettings().appendText(
+            """
             include("a", "b")
-        """)
+        """
```
Comment (bamboo) [inline subprojects/kotlin-dsl/src/integTest/kotlin/org/gradle/kotlin/dsl/resolver/KotlinScriptDependenciesResolverTest.kt:310]: ❌  indentation looks off

File: subprojects/kotlin-dsl/src/integTest/kotlin/org/gradle/kotlin/dsl/resolver/KotlinScriptDependenciesResolverTest.kt [hunk 30]
```diff
@@ -275,9 +284,11 @@ class KotlinScriptDependenciesResolverTest : AbstractKotlinIntegrationTest() {
         // because the IDE already provides user feedback for those
         assumeNonEmbeddedGradleExecuter()
 
-        val editedScript = withBuildScript("""
+        val editedScript = withBuildScript(
+            """
             doNotExists()
-        """)
+        """
```
Comment (bamboo) [inline subprojects/kotlin-dsl/src/integTest/kotlin/org/gradle/kotlin/dsl/resolver/KotlinScriptDependenciesResolverTest.kt:290]: ❌  indentation looks off

File: subprojects/kotlin-dsl/src/integTest/kotlin/org/gradle/kotlin/dsl/resolver/KotlinScriptDependenciesResolverTest.kt [hunk 31]
```diff
@@ -230,9 +236,12 @@ class KotlinScriptDependenciesResolverTest : AbstractKotlinIntegrationTest() {
         // thus disabling syntax highlighting
 
         withKotlinBuildSrc()
-        withFile("buildSrc/src/main/kotlin/Foo.kt", """
+        withFile(
+            "buildSrc/src/main/kotlin/Foo.kt",
+            """
             BOOM
-        """)
+        """
```
Comment (bamboo) [inline subprojects/kotlin-dsl/src/integTest/kotlin/org/gradle/kotlin/dsl/resolver/KotlinScriptDependenciesResolverTest.kt:243]: ❌  indentation looks off

File: subprojects/kotlin-dsl/src/integTest/kotlin/org/gradle/kotlin/dsl/resolver/KotlinScriptDependenciesResolverTest.kt [hunk 32]
```diff
@@ -176,12 +180,14 @@ class KotlinScriptDependenciesResolverTest : AbstractKotlinIntegrationTest() {
...
         assertSucceeds(
-            withBuildScript("""
+            withBuildScript(
+                """
                 require(System.getProperty("myJvmSysProp") == "systemValue") { "gradleJvmOptions" }
                 require(System.getProperty("myGradleSysProp") == "systemValue") { "gradleOptions system property" }
                 require(findProperty("myGradleProp") == "gradleValue") { "gradleOptions Gradle property" }
                 require(System.getenv("myEnvVar") == "envValue") { "gradleEnvironmentVariables" }
-            """),
+            """
```
Comment (bamboo) [inline subprojects/kotlin-dsl/src/integTest/kotlin/org/gradle/kotlin/dsl/resolver/KotlinScriptDependenciesResolverTest.kt:189]: ❌  indentation looks off

File: subprojects/kotlin-dsl/src/integTest/kotlin/org/gradle/kotlin/dsl/resolver/KotlinScriptDependenciesResolverTest.kt [hunk 33]
```diff
@@ -146,11 +148,13 @@ class KotlinScriptDependenciesResolverTest : AbstractKotlinIntegrationTest() {
...
         withKotlinBuildSrc()
 
-        withBuildScript("""
+        withBuildScript(
+            """
             plugins {
                 id("my-plugin")
             }
-        """)
+        """
```
Comment (bamboo) [inline subprojects/kotlin-dsl/src/integTest/kotlin/org/gradle/kotlin/dsl/resolver/KotlinScriptDependenciesResolverTest.kt:156]: ❌  indentation looks off

File: subprojects/kotlin-dsl/src/integTest/kotlin/org/gradle/kotlin/dsl/resolver/KotlinScriptDependenciesResolverTest.kt [hunk 34]
```diff
@@ -130,9 +130,11 @@ class KotlinScriptDependenciesResolverTest : AbstractKotlinIntegrationTest() {
 
         withKotlinBuildSrc()
 
-        withDefaultSettings().appendText("""
+        withDefaultSettings().appendText(
+            """
             apply(plugin = "my-plugin")
-        """)
+        """
```
Comment (bamboo) [inline subprojects/kotlin-dsl/src/integTest/kotlin/org/gradle/kotlin/dsl/resolver/KotlinScriptDependenciesResolverTest.kt:136]: ❌  indentation looks off

File: subprojects/kotlin-dsl/src/integTest/kotlin/org/gradle/kotlin/dsl/integration/TestKitIntegrationTest.kt [hunk 35]
```diff
@@ -127,9 +134,11 @@ class TestKitIntegrationTest : AbstractKotlinIntegrationTest() {
 
                 @Rule @JvmField val temporaryFolder = TemporaryFolder()
             }
-        """)
+        """
```
Comment (bamboo) [inline subprojects/kotlin-dsl/src/integTest/kotlin/org/gradle/kotlin/dsl/integration/TestKitIntegrationTest.kt:137]: ❌  indentation looks off

**Comment by eskatos** [comment]
@bot-gradle test this

@bamboo I did a pass over the indentation of triple quoted function parameters and fixed the code generated by the kotlin-dsl. Please take another look

**Comment by eskatos** [comment]
@bot-gradle test this

**Review by bamboo — APPROVED** [review]

**Comment by bamboo** [comment]
@bot-gradle test this

**Pull request merged**